.github/scripts/identify_removable_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+PASSED_TESTS="$(pwd)/passed_tests.txt"
+CURRENT_SKIP_LIST="$(pwd)/current_skip_list.txt"
+REMOVABLE_TESTS="$(pwd)/removable_tests.txt"
```

Comment:
I'm not convinced by the naming here. This skip list works as XFAIL list - when XFAIL test passes an error is returned, skipped test should pass no matter what.

This doesn't have to be addressed at this time - we can discuss it later.